From 0b6e1dc3290cfbc6c38a19bead9f39c582f54c40 Mon Sep 17 00:00:00 2001
From: Jan Friesse <jfriesse@redhat.com>
Date: Tue, 21 Feb 2012 14:30:35 +0100
Subject: [PATCH] iba: Use configured node id

Corosync was ignoring nodeid for iba transport and always used
autogenerated one.

Original patch by: Jason Dillaman <jdillama@redhat.com>
Signed-off-by: Jan Friesse <jfriesse@redhat.com>
Reviewed-by: Fabio M. Di Nitto <fdinitto@redhat.com>
(cherry picked from commit 04720649baad51d56426085209faa5580b44386c)
---
 exec/totemiba.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/exec/totemiba.c b/exec/totemiba.c
index 4185f16..a3499cb 100644
--- a/exec/totemiba.c
+++ b/exec/totemiba.c
@@ -1296,6 +1296,8 @@ int totemiba_initialize (
 
 	instance->totemiba_poll_handle = poll_handle;
 
+	instance->totem_interface->bindnet.nodeid = totem_config->node_id;
+
 	instance->totemiba_deliver_fn = deliver_fn;
 
 	instance->totemiba_target_set_completed = target_set_completed;
-- 
1.7.1

